vue-frontend/src/components/portfolio/VideoSection.vue

```diff
@@ -165,9 +165,13 @@
           },
         }"
         @slideChange="onSlideChange"
-        class="swiper-container"
+        class="swiper-container separate-swiper"
```

Comment:
Shouldn't we use `swiper-container` ?

Reply:
swiper-conatiner is common class in lifeatcanopas and videosection page tha's why I use new custom class to separate both swiper

Reply:
Isn't this css is scoped?

Reply:
Nope. Because when i scoped the style swiper was not displaying , I used nested CSS  here

Reply:
Ok name it `video-swiper`